conda/cli/main_config.py

```diff
@@ -563,20 +630,46 @@ def execute_config(args, parser):
 
     if args.show is not None:
         if args.show:
-            paramater_names = args.show
+            provided_parameters = tuple(
+                name for name in args.show if not name.startswith("plugins.")
+            )
+            provided_plugin_parameters = tuple(
+                name.replace("plugins.", "")
+                for name in args.show
+                if name.startswith("plugins.")
+            )
             all_names = context.list_parameters()
-            not_params = set(paramater_names) - set(all_names)
-            if not_params:
+            all_plugin_names = context.plugins.list_parameters()
+            not_params = set(provided_parameters) - set(all_names)
+            not_plugin_params = set(provided_plugin_parameters) - set(all_plugin_names)
+
+            if not_params or not_plugin_params:
                 from ..common.io import dashlist
                 from ..exceptions import ArgumentError
 
+                error_params = not_params | not_plugin_params
                 raise ArgumentError(
-                    f"Invalid configuration parameters: {dashlist(not_params)}"
+                    f"Invalid configuration parameters: {dashlist(error_params)}"
```

Comment:
Isn't this list of `error_params` potentially confusing? Since we removed the `plugins.` prefix above wont it have non-plugin params and plugin params all jumbled together?